### PR TITLE
WIP: edX IDA Template

### PR DESCRIPTION
**Pull request by clintonb**
TODO
- [x] Determine why coverage doesn't work
- [x] Add django-waffle
- [x] Add Django REST Framework
- [x] Load production settings from YAML

Future
- [ ] Make asset pipeline optional
- [ ] Make edx-pylint Python 3-compatible

### CODE REVIEW

**Comment by nedbat** [comment]
It would be great if we could use edx-lint from the start to keep pylintrc files in sync.
Also, I can help with why coverage doesn't work, if you need.

File: {{cookiecutter.repo_name}}/requirements/test.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+coverage
+django-dynamic-fixture
+django-nose
+mock
+nose-ignore-docstring
+pep8
+pylint
+
+# TODO Re-enable when Python 3 is supported.
```
Comment (nedbat) [inline {{cookiecutter.repo_name}}/requirements/test.txt]: ?? Is this repo for Python 3?

File: {{cookiecutter.repo_name}}/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - directories:
+    - $HOME/.cache/pip
+before_install:
+    - "export DISPLAY=:99.0"
+    - "sh -e /etc/init.d/xvfb start"
+install:
+    - pip install -U pip wheel
+    - pip install -r requirements/test.txt
+    - pip install coveralls
+    - make requirements_js
```
Comment (clintonb) [inline {{cookiecutter.repo_name}}/.travis.yml]: Remove.

**Comment by clintonb** [comment]
Please review https://github.com/edx/cookiecutter-django-ida/pull/1.

**Pull request closed without merging**